### PR TITLE
upgrade to google-auto-auth@0.7.1

### PR DESCRIPTION
**Pull request by ofrobots**
Brings in https://github.com/stephenplusplus/google-auto-auth/pull/21.

### CODE REVIEW

File: packages/resource/package.json [hunk 1]
```diff
@@ -57,7 +57,6 @@
   "devDependencies": {
     "arrify": "^1.0.0",
     "async": "^2.1.2",
-    "google-auto-auth": "^0.5.2",
```
Comment (stephenplusplus) [inline packages/resource/package.json]: Still need this! https://github.com/ofrobots/google-cloud-node/blob/aa5f008e1ebf8106877e23503b4ae508d20a86f7/packages/resource/system-test/resource.js#L22

Although, that can probably be re-written in the future at some point.

Comment (ofrobots) [inline packages/resource/package.json]: My bad. Too used to grepping only in the `src/` and `test/`.

Comment (ofrobots) [inline packages/resource/package.json]: Fixed.

**Comment by ofrobots** [comment]
Thanks for merging! Could we have new releases on Monday?

**Comment by stephenplusplus** [comment]
I will shoot for that 🌠

**Pull request merged**